docs/src/prototypes/CopyToClipboard/Notification.tsx

```diff
@@ -0,0 +1,24 @@
+import { Portal, createComponent } from '@stardust-ui/react'
+import * as React from 'react'
+
+type NotificationProps = {
+  children?: React.ReactNode
+  open?: boolean
+}
+
+const Notification = createComponent<NotificationProps>({
```

Comment:
Consider use case when CopyToClipboard is in a Menu as a MenuItem. Create example prototype to test it. Rename to `NotificationProvider`.

Reply:
Discuss or pair with Shift...